packages/dmn-js-shared/src/base/Manager.js

```diff
@@ -395,7 +423,8 @@ export default class Manager {
       }) || this._getInitialView(newViews);
 
       if (!newActiveView) {
-        return this._switchView(null);
+        this._switchView(null);
+        return;
```

Comment:
Let's not forget for the `importXML` migration that this schedules a new Promise, but since we don't return it, one cannot really `await` it.

My suggestion: Don't change anything right now -> tide this up in the `importXML` PR.